src/modules/FioAddress/components/ConnectWallets.tsx

```diff
@@ -197,6 +197,7 @@ class ConnectWallets extends React.Component<Props, LocalState> {
             {walletItems && Object.keys(walletItems).length ? (
               <FlashList
                 data={Object.values(walletItems)}
+                extraData={{ ...this.state.connectWalletsMap, ...this.state.disconnectWalletsMap }}
```

Comment:
This got fixed in the other PR, so I guess this commit will get dropped in the rebase.